spec/models/update_user_spec.rb

```diff
@@ -45,10 +46,44 @@
     let(:user_id) { user.id }
     let(:user) { current_user }
 
+    context 'user updates DUNS to invalid DUNS number' do
+      it 'raises validation error' do
+        params = ActionController::Parameters.new(
+          id: user_id,
+          user: {
+            duns_number: "BAD"
+          }
+        )
+
+        updater = UpdateUser.new(params, current_user)
+        updater.save
+
+        expect(updater.errors).to eq(
+          'DUNS number format is invalid'
+        )
+      end
```

Comment:
ah wait -- this is not hitting the SAM API -- we are validating the format. So we are not actually saving this new duns when someone submits it (we keep old value until they submit a potentially valid duns number, at which point we hit the API.

added spec expectation to clarify this behavior at 20b605d166498e1aace44de2668bd342f72e5103